ios-base/Common/Models/AuthViewModelStateDelegate.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AuthViewModelStateDelegate.swift
+//  ios-base
+//
+//  Created by German on 5/20/20.
+//  Copyright © 2020 Rootstrap Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol NetworkStatusDelegate: class {
```

Comment:
Shouldn’t this be defined in a file of it's own along with it's extension? Since this is a general protocol for networking updates not related to auth particularly